### PR TITLE
Dtspb 90 intestacy e2 e tests

### PR DESCRIPTION
**Pull request by phaniperla**
**Before creating a pull request make sure that:**

- [ X] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-90

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

**Review by isha2040 — APPROVED** [review]

File: test/end-to-end/pages/steps.js [hunk 1]
```diff
@@ -79,6 +79,7 @@ module.exports = () => {
 
         //Equality and diversity page
         exitEqualityAndDiversity: steps.equalityanddiversity.startpage,
+        completePCQ: steps.equalityanddiversity.pcqstartpage,
```
Comment (meganaslattery) [inline test/end-to-end/pages/steps.js:82]: What's the difference between startpage and pcqstartpage?

File: test/end-to-end/paths/intestacy/intestacy.en.test.js [hunk 2]
```diff
@@ -80,7 +80,10 @@ Scenario(TestConfigurator.idamInUseText('Intestacy Child Journey - Digital iht,
     I.enterApplicantName('ApplicantFirstName', 'ApplicantLastName');
     I.enterApplicantPhone();
     I.enterAddressManually();
-    I.seeSummaryPage('*');
+    if (TestConfigurator.equalityAndDiversityEnabled()) {
+        I.exitEqualityAndDiversity();
+        I.completePCQ();
```
Comment (meganaslattery) [inline test/end-to-end/paths/intestacy/intestacy.en.test.js:85]: Why do you need both of these steps?

Comment (phaniperla) [reply test/end-to-end/paths/intestacy/intestacy.en.test.js:85]: We need both methods as the first one performing back button navigation and completePCQ() method performing click action of "Save and continue" button.

File: test/end-to-end/pages/steps.js (hunk 1 above)
Comment (phaniperla) [reply test/end-to-end/pages/steps.js:82]: We need both methods as the first one performing back button navigation and completePCQ() method performing click action of "Save and continue" button.

**Review by isha2040 — APPROVED** [review]

**Review by meganaslattery — APPROVED** [review]

**Pull request merged**